### DISCUSSION TITLE
Best settings for a rtx 4080 with 16gb vram to use Lora?

### DISCUSSION THREAD

**Discussion by elen07zz** [lllyasviel/stable-diffusion-webui-forge#1467] [General]
Hello, I need some help to configure forge, I remember be able to get speeds around 1.1 s/it or something similar, But now i'm getting speeds around 1.7s/it or 2s/it.
But not only that, I'm also using an absurd amount of ram now, today I saw that I was using 50gb of ram! That's really absurd!

I'm not sure what i'm doing wrong. 

Could anyone who has a GPU with a similar amount of VRAM share their settings or tips?

**Comment by elen07zz** [comment]
help

**Comment by stripealipe** [comment]
We need more info mate. What model are you using etc? Post a screen of your settings.

**Comment by TimmekHW** [comment]
Flux, RTX 4090:
![image](https://github.com/user-attachments/assets/5057745e-0632-4ca4-a54f-3fd1945e0ac8)
Astronaut in a jungle, cold color palette, muted colors, very detailed, sharp focus
Steps: 20, Sampler: Euler, Schedule type: Simple, CFG scale: 1, Distilled CFG Scale: 3.5, Seed: 12345, Size: 896x1152, Model hash: 06f96f89f6, Model: flux_dev, Version: f2.0.1v1.10.1-previous-417-g8ff0dc6f, Module 1: ae, Module 2: clip_l, Module 3: t5xxl_fp8_e4m3fn

Time taken: 40.2 sec.

A: 13.38 GB, R: 13.97 GB, Sys: 15.6/23.9883 GB (65.1%)
![image](https://github.com/user-attachments/assets/f6bdb52c-3e92-467f-9a84-51da06910ef4)


set COMMANDLINE_ARGS= --api --cuda-malloc --cuda-stream --pin-shared-memory --disable-xformers --opt-sdp-attention --listen
